### DISCUSSION TITLE
Email notification on Success backup

### DISCUSSION THREAD

**Discussion by quath2021** [erikw/restic-automatic-backup-scheduler#106] [Q&A] [answered]
First I wanted to thank everyone for the work that has gone into this. I have found it very helpful and learnt a lot.

In the documentation it talk about Failure notifications. Is there a way to enable Success notifications as well please?

Thanks

**Comment by erikw** [comment, marked as answer]
Hello, great to hear that you're having fun and learning!

The failure email uses the systemd service unit feature `OnFailure`:
https://github.com/erikw/restic-automatic-backup-scheduler#optional-email-notification-on-failure



In the documentation

https://www.freedesktop.org/software/systemd/man/systemd.unit.html

I see there's also a that there's also a `OnSuccess` . Thus I think you could probably easily achieve this. Try setting up the failure notification successfully first, then simply add an individual line in the service file using `OnSuccess` instead and see what happens.

Good luck!
